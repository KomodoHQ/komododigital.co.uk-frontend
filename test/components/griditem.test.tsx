import React from 'react';
import GridItem from '../../src/components/griditem';
import renderer from 'react-test-renderer';

test('GridItem element is rendered', () => {
  const title = "title";
  const image =  "https://picsum.photos/100/?random";
  const children = <p></p>;
  const component = renderer.create(<GridItem title={title} image={image}>{children}</GridItem>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
