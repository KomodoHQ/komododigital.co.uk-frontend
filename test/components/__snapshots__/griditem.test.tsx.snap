// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridItem element is rendered 1`] = `
<div>
  <div>
    IMAGE
  </div>
  <h3 />
  <p />
</div>
`;
